python/nano/test/pytorch/tests/test_scale_lr.py

```diff
@@ -0,0 +1,225 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+
+import pytest
+import math
+from unittest import TestCase
+
+from test.pytorch.utils._train_torch_lightning import create_data_loader, data_transform
+from test.pytorch.utils._train_torch_lightning import create_test_data_loader
+from bigdl.nano.pytorch.vision.models import vision
+from bigdl.nano.pytorch import Trainer
+from bigdl.nano.pytorch.utils import TORCH_VERSION_LESS_1_10
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import LearningRateMonitor
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+num_classes = 10
+batch_size = 32
+dataset_size = 256
+num_workers = 0
+data_dir = os.path.join(os.path.dirname(__file__), "../data")
+
+
+class Resnet_2_0(pl.LightningModule):
```

Comment:
What's the difference between `Resnet_2_0` and `Resnet_2_2`? Is it possible to use more meaningful names?

Reply:
Resnet_2_0 used to test the optimizers' lr scaling and warmup has two optimizers and no lr_scheduler.
And Resnet_2_2 used to test lr_schedulers' base_lrs scaling

Reply:
I'm not sure, maybe `Resnet_opt` and `Resnet_opt_sch`?

Reply:
`ResNetWith2Optimzers` and `ResNetWithScheduler`?

Reply:
ok, these are better.